packages/sdk/src/subgraph.ts

```diff
@@ -95,10 +95,10 @@ export const createSubgraphEvts = (): {
 };
 
 export type ActiveTransaction = {
-  txData: TransactionData;
+  crosschainTx: CrosschainTransactionData;
   status: NxtpSdkEvent;
   bidSignature: string;
-  caller: string;
+  caller: string; // TODO: which caller tho?
```

Comment:
This doesn't make sense. There could be a receiver prepare caller, a sender/receiver fulfill caller, or a sender/receiver cancel caller. We should simply remove this from the type, or make it more specific. But as it stands this is very ambiguous

Reply:
I think we can check the caller side from status, but we should surely make it clearer.
if we update it with every updated transaction status or if want to track each method(fulfill/cancel) caller on both sides.